src/test/kotlin/de/jodamob/kotlin/testrunner/sample/TestWithRunner.kt

```diff
@@ -1,15 +1,17 @@
 package de.jodamob.kotlin.testrunner.sample
 
 import de.jodamob.kotlin.testrunner.KotlinTestRunner
+import de.jodamob.kotlin.testrunner.OpenedClasses
+import de.jodamob.kotlin.testrunner.nonopen.FinalNonOpenClassSample
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mockito
 import org.mockito.exceptions.base.MockitoException
 
 @RunWith(KotlinTestRunner::class)
+@OpenedClasses("de.jodamob.kotlin.testrunner.sample")
```

Comment:
In current case you can both:

```
@OpenedClasses("de.jodamob.kotlin.testrunner.sample")
```
and
```
@OpenedClasses("de.jodamob.kotlin.testrunner.sample.ConcreteClass")
```

as this is only a prefix.

Inheriting package is a fancy feature but in many cases it will be to narrow. Many times I want to mock classes that are external dependencies and are in different package. What do you think?